gover/utils/recorder.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/yezihack/colorlog"
+)
+
+type messagePack struct {
+	ProtoName string  `json:"protoName"`
+	Object    Message `json:"object"`
+}
+
+type Recorder struct {
+	q *BytesQueue
+	h *PacketHandler
+	d []*messagePack
+}
+
+func (r *Recorder) Stop() {
+	r.q.Close()
+	content, err := json.Marshal(r.d)
```

Comment:
replace with `content, err := json.MarshalIndent(r.d, "", "  ")` for json pretty print

Reply:
Cool, I will be replacing it with [protojson](https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson), the default JSON marshaler's output is quite different from the python version, like the JSON key serializes with snake_case but not camelCase, enum using number not a string, etc...

Reply:
Ahh ok, I thought that was caused by protobuf go